distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Postgres/Postgres_Dialect.enso

```diff
@@ -267,7 +267,7 @@ make_order_descriptor internal_column sort_direction text_ordering =
             ## In the future we can modify this error to suggest using a custom defined collation.
             if text_ordering.sort_digits_as_numbers then Error.throw (Unsupported_Database_Operation.Error "Natural ordering is currently not supported. You may need to materialize the Table to perform this operation.") else
                 case text_ordering.case_sensitivity of
-                    Nothing ->
+                    Case_Sensitivity.Default ->
```

Comment:
I'm so fond of this change! Meaningful constructors are always much better over `Nothing` carrying thousands of meanings.